tests/unit/test_irc.py

```diff
@@ -351,27 +340,32 @@ def test_configure_evaluates_configuration_file_local(irc_bridge_service, mocker
         mocker.call(f"{IRC_BRIDGE_CONFIG_FILE_PATH.absolute()}", "w", encoding="utf-8"),
     ]
 
-    mock_open.assert_has_calls(calls, any_order=True)
+    mock_builtin_open.assert_has_calls(calls, any_order=True)
```

Comment:
I think we should tmp_path or any similar alternative instead of mocking open

Reply:
Keeping the discussion here.

Nice input. I rewrote the test_prepare_installs_snap_package_and_creates_configuration_files. Please check if it's what you had in mind so I can apply for others as well.